File: 프로그래머스/2/42578. 의상/의상.java
import java.util.*;

class Solution {
    public int solution(String[][] clothes) {
        HashMap<String, Integer> cloth = new HashMap<>();
        for(String[] c: clothes) {
            String k = c[1];
            String v = c[0];
            
            if (cloth.containsKey(k)) {
                cloth.put(k, cloth.get(k) + 1);
            } else {
                cloth.put(k, 1);
            }
        }
        
        System.out.println(cloth);
        int answer = 1;
        for (int value : cloth.values()) {
            answer *= (value + 1);
        }
        
        
        return answer - 1;
    }
}

File: 프로그래머스/2/42578. 의상/의상.py
def solution(clothes):
    cloth = {}
    for c in clothes:
        if c[1] in cloth:
            cloth[c[1]] += 1
        else:
            cloth[c[1]] = 1
    
    cvalues = cloth.values()
    answer = 1
    for c in cvalues:
        answer *= (c+1)
    
    return answer - 1